### PR TITLE
Adding python3-ntplib in rhel 8

### PR DESCRIPTION
**Pull request by ct2034**
Should be available according to https://centos.pkgs.org/8/centos-appstream-aarch64/python3-ntplib-0.3.3-10.el8.noarch.rpm.html

### CODE REVIEW

**Review by tfoote — APPROVED** [review]
This looks fine to me. @cottsay Do we need the extra python version logic?

**Comment by cottsay** [comment]
This shouldn't be necessary. `python%{python3_pkgversion}-ntplib` evaluates to `python3-ntplib` on RHEL 8.

**Comment by tfoote** [comment]
We just need a second review then.

**Comment by cottsay** [comment]
Let me clarify: this change shouldn't be necessary at all.

**Comment by ct2034** [comment]
I see what you mean. Without being able to verify this, I trust you and close this. 😊

**Pull request closed without merging**
